### PR TITLE
[KRYS-3351] Add discord and tele button to chart view

### PR DESCRIPTION
**Pull request by MrQKyber**

### CODE REVIEW

File: KyberNetwork/Models/Overview.swift [hunk 1]
```diff
@@ -61,6 +61,9 @@ struct TokenDetailInfo: Codable {
 struct Links: Codable {
     let homepage: String
     let twitterScreenName: String
+  let discord: String
```
Comment (tungnguyen20) [inline KyberNetwork/Models/Overview.swift:64]: Please fix this indent

File: KyberNetwork/KyberNetwork/Screens/Tabbars/Overview/Controllers/ChartViewController.swift [hunk 2]
```diff
@@ -310,6 +310,8 @@ enum ChartViewEvent {
   case openTwitter(name: String)
   case getPoolList(address: String, chainId: Int)
   case selectPool(source: String, quote: String)
+  case openDiscord(name: String)
+  case openTelegram(name: String)
```
Comment (tungnguyen20) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/Overview/Controllers/ChartViewController.swift:314]: `name` must be `url`

File: KyberNetwork/KyberNetwork/Screens/Tabbars/Overview/Coordinator/OverviewCoordinator.swift [hunk 3]
```diff
@@ -363,6 +363,10 @@ extension OverviewCoordinator: ChartViewControllerDelegate {
       self.openCommunityURL("https://twitter.com/\(name)/")
     case .selectPool(source: let source, quote: let quote):
       break
+    case .openDiscord(link: let link):
```
Comment (tungnguyen20) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/Overview/Coordinator/OverviewCoordinator.swift:366]: `.openDiscord(let link)` is enough

Comment (MrQKyber) [reply KyberNetwork/KyberNetwork/Screens/Tabbars/Overview/Coordinator/OverviewCoordinator.swift:366]: It's okay with this format. Plz ignore it

**Review by tungnguyen20 — APPROVED** [review]

**Pull request merged**